lineman/app/components/group_page/group_page_controller.coffee

```diff
@@ -38,7 +38,7 @@ angular.module('loomioApp').controller 'GroupPageController', ($rootScope, $loca
     ModalService.open LogoPhotoForm, group: => @group
 
   @handleSubscriptionSuccess = ->
-    if AppConfig.chargify and $location.search().chargify_success?
+    if (AppConfig.chargify or AppConfig.environment != 'production') and $location.search().chargify_success?
```

Comment:
Not sure I understand? For testing in dev we could set `.env` to contain the chargify test creds, for automated testing we could stub out `AppConfig.chargify = true`?